package.json

```diff
@@ -52,7 +52,7 @@
     "lint-staged": "^13.0.4",
     "prettier": "^1.17.1",
     "ts-jest": "^29.0.0",
-    "typedoc": "^0.23.21",
+    "typedoc": "~0.25.13",
```

Comment:
This is an optional update for this PR and actually breaks CI because we support Node.js 14 in this package.

Reply:
downgraded to 0.24 in [fe2dfa6](https://github.com/electron/get/pull/288/commits/fe2dfa652a272c84e55cb1ed19c02240641689b5)

Reply:
this is still incompatible because Node doesn't have v14 binaries for arm64, so we hacked CI to use Node 14.16... and `typedoc 0.24` requires Node.js >= 14.17.

I think there are actually a lot of features in TypeDoc 0.24 that we need. :(

Reply:
As discussed over video call earlier this week, `--ignore-engines` applied for Node.js 14 CI.